meta-mender-overo-sdcard/conf/layer.conf

```diff
@@ -0,0 +1,13 @@
+# Board specific layer configuration for meta-mender
+# Copyright 2018 Northern.tech AS
+
+# We have a conf and classes directory, add to BBPATH
+BBPATH .= ":${LAYERDIR}"
+
+# We have recipes-* directories, add to BBFILES
+BBFILES += "${LAYERDIR}/recipes-*/*/*.bb \
+	${LAYERDIR}/recipes-*/*/*.bbappend"
+
+BBFILE_COLLECTIONS += "mender-overo"
+BBFILE_PATTERN_mender-overo = "^${LAYERDIR}/"
+BBFILE_PRIORITY_mender-overo = "9"
```

Comment:
Depends on meta-overo?

Reply:
I can do that but we don't have any LAYERDEPENDS in the morty branch at all.  @kacf preference?

Reply:
No need here then, but something to keep in mind if this is forward ported.